### PR TITLE
Created and updated docs for 1.20-17

### LINKED ISSUE #1024: Update docs for 1.20-17

### PR DESCRIPTION
**Pull request by kschumy**
Created and updated docs for 1.20-17

closes #1024

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/contents/index.md [hunk 1]
```diff
@@ -148,6 +148,7 @@ if you are interested in the AL2 container runtime kernel version.
 * [v1-21-eks-1](releases/1-21/1/index.md) (July 19, 2021)
 
 #### EKS-D 1.20 Version Dependencies
+* [v1-20-eks-17](releases/1-20/17/index.md) (June 21, 2022)
```
Comment (kschumy) [inline docs/contents/index.md:151]:
```suggestion
* [v1-20-eks-17](releases/1-20/17/index.md) (June 20, 2022)
```

File: docs/contents/releases/1-20/17/release-announcement.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+Amazon EKS Distro v1.20-17 is now available. This release includes a new Kubernetes patch version, a new patch to Kubernetes, an update to Amazon Linux 2 that contains the latest security fixes, and updates to some project versions and project dependencies. Amazon EKS Distro v1.20-17 builds are available through ECR Public Gallery (https://gallery.ecr.aws/eks-distro) and GitHub (https://github.com/aws/eks-distro)
```
Comment (kschumy) [inline docs/contents/releases/1-20/17/release-announcement.txt:1]:
```suggestion
Amazon EKS Distro v1.20-17 is now available. This release includes a new patch to Kubernetes, an update to Amazon Linux 2 that contains the latest security fixes, and updates to some project versions and project dependencies. Amazon EKS Distro v1.20-17 builds are available through ECR Public Gallery (https://gallery.ecr.aws/eks-distro) and GitHub (https://github.com/aws/eks-distro)
```

**Review by TerryHowe — APPROVED** [review]
/lgtm

**Pull request merged**